docs/quick-reference.adoc

```diff
@@ -13,11 +13,14 @@ constant :: 3.14
 variable := []
 two: int : 2
 text: str = "abc"
-text = "A short sentence."  // <1>
-----
 
-<1> A single `=` is used to update the value of a variable. Variables are
-declarared with `:=`.
+num := .5         // Now 0.5
+num = 2.0         // Now 2.0, note the lack of ":"
+num += 1.0        // Now 3.0
+num -= 2.0        // Now 1.0
+num *= 10.        // Now 10.0
+num /= 2.         // Now 5.0
```

Comment:
Maybe something like `num => 0.5`? I don't really like the word 'now'... It's wierd...